TestAssets/TestProjects/SimpleDependencies/SimpleDependencies.csproj

```diff
@@ -11,9 +11,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETCore.App">
-      <Version>1.0.1</Version>
-    </PackageReference>
```

Comment:
Did you leave any tests with explicit package references overriding the default?